### ISSUE TITLE
[Issue] GROUP_SUBSYSTEMS not-working for No-Intro's Non-Redump - Sony - Playstation Portable

### ISSUE THREAD

**Issue by monyarm** [alucryd/oxyromon#94]
With GROUP_SUBSYSTEMS set to true, csos in the `Non-Redump - Sony - Playstation Portable` dat, don't go to `Sony - Playstation Portable` as they should, but instead to `Non-Redump - Sony - Playstation Portable`.

I'm using oxyromon 0.16.0, on Gentoo Linux.

**Comment by alucryd** [comment]
Yeah, grouping subsystems only works for suffixes right now, not prefixes. I don't know why no-intro went for a prefix in this case, but in order to make that one work I'll need to handle it as a special case.

**Comment by leo60228** [comment]
No-Intro's "Redump Custom DATs" use yet another format (`Nintendo - Wii U - WUX`). I'm not sure if there's a good way to handle them without special-casing.

**Comment by leo60228** [comment]
Arguably, the need for them should be sidestepped by adding WUX support to oxyromon, though. I might do that, but the WUX CLI tool doesn't have macOS/Linux support or a listed license.

**Comment by alucryd** [comment]
Wii U is on my TODO list indeed :) I dumped my games as WUA a while ago, and I'm definitely interested in supporting WUX. If the WUX format isn't too complicated, making a rust lib for it could be a fun weekend project.

**Comment by leo60228** [comment]
> If the WUX format isn't too complicated, making a rust lib for it could be a fun weekend project.

Decided to work on this, currently only decompression is implemented: https://docs.rs/wux (along with a CLI: https://crates.io/crates/wux-cli)
